pyccl/halos/pk_4pt.py

```diff
@@ -669,6 +670,9 @@ def halomod_trispectrum_2h_22(cosmo, hmc, k, a, prof, *, prof2=None,
         p_of_k_a (:class:`~pyccl.pk2d.Pk2D`): a `Pk2D` object to
             be used as the linear matter power spectrum. If `None`, the power
             spectrum stored within `cosmo` will be used.
+        separable_growth (Boolean): Indeicates whether a separable
```

Comment:
Typo: Indeicates -> Indicates

Reply:
and Boolean -> bool for consistency (and so the docs look nice)